### PR TITLE
update text server to support compute logprobs

### PR DESCRIPTION
**Pull request by Zhilin123**
# What does this PR do ?

update text server to support compute logprobs

**Collection**: [nlp]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [x] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: nemo/collections/nlp/modules/common/text_generation_server.py [hunk 1]
```diff
@@ -192,7 +199,7 @@ def put(self):
                 repetition_penalty,
                 end_strings=end_strings,
                 min_tokens_to_generate=min_tokens_to_generate,
-                **extra,
+                compute_logprob=compute_logprob ** extra,
```
Comment (yidong72) [inline nemo/collections/nlp/modules/common/text_generation_server.py:202]: there is a comma in between `compute_logprob=compute_logprob` and `** extra,`

Comment (yidong72) [reply nemo/collections/nlp/modules/common/text_generation_server.py:202]: remember run `python setup.py style --fix`

Comment (Zhilin123) [reply nemo/collections/nlp/modules/common/text_generation_server.py]: thanks

**Review by yidong72 — APPROVED** [review]
LGTM

**Pull request merged**